integration/test/performancetest/performance_test.go

```diff
@@ -42,18 +64,44 @@ func TestPerformance(t *testing.T) {
 		t.Fatalf("Error: %v", err)
 	}
 
-	//------Placeholder to put data into database------//
-	//useless code so data get used and compiler isn't mad
 	if data == nil {
 		t.Fatalf("No data")
 	}
-	//data base 
+
+	//data base
```

Comment:
Didn't notice this. Normally you'd initialize these connections at the beginning of the function

Reply:
Moved to beginning